aws-lambda-java-tests/src/test/java/com/amazonaws/services/lambda/runtime/tests/EventLoaderTest.java

```diff
@@ -26,7 +26,6 @@ public void testLoadApiGatewayRestEvent() {
 
         assertThat(event).isNotNull();
         assertThat(event.getBody()).isEqualTo("Hello from Lambda!");
-        assertThat(event.getHeaders()).hasSize(2);
```

Comment:
Why remove this? Can we not include an equivalent assertion for `HttpHeaders`?

Reply:
I think I removed this because I created a new test class dedicated solely to testing headers in `aws-lambda-java-tests/src/test/java/com/amazonaws/services/lambda/runtime/tests/HeadersTest.java`, and the header check here did not seem relevant to the name of this set of tests `EventLoaderTest`. At least that's the reasoning that would make sense to me right now.

But since later classes in here do the header test too, right now I'd think there's no real benefit to removing it either. So I'll add it back!